fetcher/block_test.go

```diff
@@ -91,42 +150,78 @@ func TestBlockRetry(t *testing.T) {
 	for name, test := range tests {
 		t.Run(name, func(t *testing.T) {
 			var (
-				tries       = 0
-				assert      = assert.New(t)
-				ctx, cancel = context.WithCancel(context.Background())
-				endpoint    = "/block"
+				tries            = 0
```

Comment:
how about rename for clarity?
```suggestion
				blockTries            = 0
```